.github/workflows/examples.yml

```
@@ -61,7 +61,7 @@ jobs:
         path: vendor/bundle
         key: package-app-gem-cache-${{ hashFiles('react_on_rails.gemspec') }}-${{ hashFiles('Gemfile.development_dependencies') }}-${{ matrix.versions }}
     - id: get-sha
-      run: echo "::set-output name=sha::$(git rev-parse HEAD)"
+      run: echo "sha=$(git rev-parse HEAD)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)